src/modules/iotjs_module_buffer.c

```diff
@@ -82,6 +82,24 @@ size_t iotjs_bufferwrap_length(iotjs_bufferwrap_t* bufferwrap) {
 }
 
 
+bool iotjs_jbuffer_is_bufferwrap(const jerry_value_t jbuffer,
+                                 iotjs_bufferwrap_t** bufferwrap) {
+  if (jerry_value_is_object(jbuffer)) {
```

Comment:
No need to do extra check if the `jbuffer` is an object or not, the native pointer call will handle it.